TerminalDocs/command-palette.md

```diff
@@ -121,7 +121,7 @@ The above command would behave like the following three commands:
             "command": { "action": "newTab", "profile": "Command Prompt" }
         },
         {
-            "icon": "C:\\path\\to\\icon",
+            "icon": "C:\\path\\to\\icon.png",
```

Comment:
```suggestion
            "icon": "C:\\path\\to\\icon",
```
I wouldn't recommend making this change, because the icon field can take more file types than just a .png file.